components/emoji_picker/emoji_picker.tsx

```diff
@@ -21,7 +22,7 @@ import EmojiPickerCategories from 'components/emoji_picker/components/emoji_pick
 import EmojiPickerCustomEmojiButton from 'components/emoji_picker/components/emoji_picker_custom_emoji_button';
 import EmojiPickerCurrentResults from 'components/emoji_picker/components/emoji_picker_current_results';
 
-import type {PropsFromRedux} from './index';
```

Comment:
We keep the import as type to other line because it will be removed from final code and not cause circular dependency

Reply:
Yes, you are right. I reverted it back. This was the reason of failing tests